### ISSUE TITLE
Refine functional/non-functional requirements

### ISSUE THREAD

**Issue by WesMDonald** [101josh10/Radiology-Information-System#7]

**Comment by 101josh10** [comment]
Just added scheduling requirements. I imagine this will pretty much stay in the In Progress column, seeing as how we will have to add/change requirements as we go along.

**Comment by WesMDonald** [comment]
Agreed.  We might work on this till the final version of the project
